conversations/v2/webhook-events.yaml

```
@@ -771,6 +872,48 @@ components:
           type: string
           description: Description of the interactive item clicked
           example: This is the description of a list item
+    Order:
+      type: object
+      description: Represents an order sent from a channel
+      required:
+        - items
+      properties:
+        catalogId:
+          type: string
+          description: ID of the catalog the items origin from
+          example: 123980-12313123
+        text:
+          type: string
+          description: Optional message from the customer
+          example: Hi, please make sure it's delivered on the afternoon
+        items:
+          type: array
+          description: Products placed into the basket for this order
```

```suggestion
          description: Products placed into the shopping cart for this order
```
Just to use the same term we are using above :)